src/lib/ChargeStation/eventHandlers/ocpp-16/handle-change-configuration.js

```diff
@@ -3,8 +3,15 @@ export default async function handleChangeConfiguration({
   callMessageId,
   callMessageBody,
 }) {
-  const { key, value } = callMessageBody;
-  chargepoint.configuration[key] = value;
+  const { key } = callMessageBody;
+
+  chargepoint.configuration.setVariable(key, callMessageBody);
+
+  if (chargepoint.configuration[key]) {
+    chargepoint.configuration[key].value = value;
+  } else {
+    chargepoint.configuration[key] = callMessageBody;
+  }
```

Comment:
Why don't we place the following code into `chargepoint.configuration.setVariable(key, callMessageBody)`?
```js
  if (chargepoint.configuration[key]) {
    chargepoint.configuration[key].value = value;
  } else {
    chargepoint.configuration[key] = callMessageBody;
  }
```